components/SocialIcons/SocialSvgComponents.tsx

```diff
@@ -0,0 +1,66 @@
+import { SVGProps } from "react";
```

Comment:
Instead of placing all of these icons into one file, I would recommend creating a file for each: 

`components/svgs/{name}Icon.tsx`

I would argue this approach increases usability.